ga4gh/backend.py

```diff
@@ -574,7 +569,8 @@ def variantAnnotationsGenerator(self, request):
         compoundId = datamodel.VariantAnnotationSetCompoundId.parse(
             request.variantAnnotationSetId)
         dataset = self.getDataRepository().getDataset(compoundId.datasetId)
-        variantAnnotationSet = dataset.getVariantAnnotationSet(
+        variantSet = dataset.getVariantSet(compoundId.variantSetId)
+        variantAnnotationSet = variantSet.getVariantAnnotationSet(
```

Comment:
Fair enough @macieksmuga, but the schema doesn't actually support this at the moment. I'm just trying to simplify things right now, we can make them more complicated when the need arises.